src/Badge/src/Model/UseCase/CreateComposerLockBadge.php

```diff
@@ -56,21 +50,15 @@ public function createComposerLockBadge($repository, $format = 'svg')
             ->getRepository()
         );
 
-        $request = $this->client->head(
-            $repo . '/blob/master/composer.lock',
-            array(),
-            array(
+        $client = HttpClient::create();
```

Comment:
If you pass in a client from the framework (manually, or auto-wired), then as of Symfony 4.4, it will also show queries in the inspector/profiler.

Reply:
@alister : Thanks for reviewing. Can we use HttpClientInterface then (Symfony\Contracts\HttpClient\HttpClientInterface).